container-tests/tests/src/test/resources/testcontainers/goQuorum/qdata/geth/chaindata/CURRENT

```diff
@@ -0,0 +1 @@
+MANIFEST-000000
```

Comment:
Do we have to check in the files in chaindata and lightchaindata?

Reply:
Removed.